Sources/_StringProcessing/Engine/MEQuantify.swift

```diff
@@ -1,26 +1,37 @@
 extension Processor {
   func _doQuantifyMatch(_ payload: QuantifyPayload) -> Input.Index? {
-    var next: Input.Index?
+    // FIXME: is the below updated for scalar semantics?
```

Comment:
This method should be okay as long as it's deferring out to methods that handle the semantics correctly — let's make sure those are properly tested.